doc/Command-Reference.md

```diff
@@ -3955,9 +3962,12 @@ This command is used to configure the TPID for the Physical/PortChannel interfac
   admin@sonic:~$ sudo config interface tpid Ethernet64 0x9200
   ```
 
-**config interface breakout**
+**config interface breakout (Versions >= 202006)**
+
+This command is used to set breakout mode available for user-specified interface. Check the platform-specific port configuration file(i.e. platform.json)
+to get available breakout mode for each parent interface.Based on the platform.json and the current mode set in interface, this command acts on setting
+breakout mode for the interface.
 
-This command is used to set breakout mode available for user-specified interface.
 kindly use, double tab i.e. <tab><tab> to see the available breakout option customized for each interface provided by the user.
```

Comment:
Suggest removing "kindly use"

i.e. "Double tab <tab><tab> to see the...."

Reply:
Modified as suggested. 👍